### PR TITLE
Adding the ability to generate intervals on filtered variants as well as unfiltered variants.

### PR DESCRIPTION
**Pull request by LeeTL1220**
Default is previous behavior.

Added testing.

Note that this change may need to be made in picard as well.

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@TedBrookings Thanks!

File: src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/VcfToIntervalList.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
         programGroup = VariantProgramGroup.class
 )
 public final class VcfToIntervalList extends PicardCommandLineProgram {
+    public static final String INCLUDE_FILTERED_LONG_NAME = "includeFiltered";
```
Comment (TedBrookings) [inline src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/VcfToIntervalList.java]: <Hair-Splitting>Might be clearer to move declaration of INCLUDE_FILTERED names to just before they are first used.</Hair-Splitting>

Comment (TedBrookings) [inline src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/VcfToIntervalList.java]: Splitting hairs: might be clearer to move declaration of INCLUDE_FILTERED names to just before they are first used.

Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/VcfToIntervalList.java]: This is the convention used elsewhere in GATK.

No action.

**Comment by cmnbroad** [comment]
@LeeTL1220 Does GATK have a dependency on this change ? If so this will definitely need to be made in Picard as well since these GATK copies of the Picard tool are soon to be obsoleted.

**Comment by LeeTL1220** [comment]
@cmnbroad I put in a test in GATK that should fail if the dependency is moved to Picard.  So, you are 100% correct that these changes will need to go into Picard.

**Pull request merged**